lib/view_component/test_helpers.rb

```diff
@@ -190,6 +190,29 @@ def with_request_url(path, host: nil)
       @vc_test_controller = old_controller
     end
 
+    # Set the controller helpers to be used while executing the given block,
+    #
+    # ```ruby
+    # with_helpers(UsersHelper) do
```

Comment:
I was imagining that this could be used to mock helpers instead of including helpers that should already be present on the view context.

e.g.

```ruby
test "foo" do
  # delegates to the current_user method
  with_helpers(current_user: :current_user) do
    render_inline MyComponent.new
  end

  current_user = create(:user)
  # stubs the current_user helper to return this value
  with_helpers(current_user: current_user) do
    render_inline MyComponent.new
  end
end

def current_user
  # some logic
end
```

Reply:
@BlakeWilliams  moving this to a new pr